test/test_mobile_optimizer.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import torch
+import torch.backends.xnnpack
+from torch.utils import mobile_optimizer
+from torch.nn import functional as F
+
+FileCheck = torch._C.FileCheck
+
+class TestOptimizer(unittest.TestCase):
+
+    @unittest.skipUnless(torch.backends.xnnpack.enabled,
+                     " XNNPACK must be enabled for these tests."
+                     " Please build with USE_XNNPACK=1.")
+    def test_optimize_for_mobile(self):
+        batch_size = 2
+        input_channels_per_group = 6
+        height = 16
+        width = 16
+        output_channels_per_group = 6
+        groups = 4
+        kernel_h = kernel_w = 3
+        stride_h = stride_w = 1
+        pad_h = pad_w = 1
+        dilation = 1
+        input_channels = input_channels_per_group * groups
+        output_channels = output_channels_per_group * groups
+        kernels = (kernel_h, kernel_w)
+        strides = (stride_h, stride_w)
+        paddings = (pad_h, pad_w)
+        dilations = (dilation, dilation)
+        conv_weight_shape = (output_channels, input_channels_per_group, kernel_h, kernel_w)
+        conv_bias_shape = (output_channels)
+
+        input_data = torch.rand((batch_size, input_channels, height, width))
+        conv_weight = torch.rand((output_channels, input_channels_per_group, kernel_h, kernel_w))
+        conv_bias = torch.rand((output_channels))
+        result = F.conv2d(input_data, conv_weight, conv_bias, strides, paddings, dilations, groups)
+        weight_output_dim = 24
+        linear_input_shape = result.shape[1]
+        linear_weight_shape = (weight_output_dim, linear_input_shape)
+
+        class MyTestModule(torch.nn.Module):
+            def __init__(self, activation_fn=F.relu):
```

Comment:
Since this test is really use for relu only, lets remove activation_fn. Just called relu directly in forward.